libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -272,6 +272,12 @@ class AC_PosControl
     /// get desired roll, pitch which should be fed into stabilize controllers
     float get_roll() const { return _roll_target; }
     float get_pitch() const { return _pitch_target; }
+    float get_heading() const { return _heading; }
+    float get_rate_heading() const { return _rate_heading; }
+    Vector3f get_thrust_vector() const;
```

Comment:
Let's put a comment above get_thrust_vector() as well specifying what it is and the units.